src/components/grid-aware/TwoParagraphBlock/TwoParagraphBlock.stories.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+
+import TwoParagraphBlock from "./TwoParagraphBlock";
+import image from "./stories/darcel-jackson-headshot.png";
+
+export default {
+  title: "Grid-Aware/TwoParagraphBlock",
+  component: TwoParagraphBlock,
+};
+
+const Template = ({ title, paragraph1, paragraph2, image1, ctaButton }) => (
+  <div style={{ border: "1px dashed rebeccapurple" }}>
+    <TwoParagraphBlock
+      title={title}
+      paragraph1={paragraph1}
+      paragraph2={paragraph2}
+      image={image1}
+      ctaButton={ctaButton}
+    />
+  </div>
+);
+
+export const Gray = Template.bind({});
+Gray.args = {
+  title: "Rooted in the Community",
+  paragraph1:
+    "ShelterTech was founded in 2016 by Darcel Jackson, who experienced homelessness first hand. Darcel envisioned a Bay Area where technology serves everyone, and those who were most vulnerable had the digital infrastructure and tools they need to get back on their feet.",
+  paragraph2: (
+    <>
+      <p style={{ marginTop: "0" }}>
```

Comment:
I see that the inline style prevents a downward shift in the desktop view for this paragraph. However if another developer with no knowledge of how the view should look like may be unaware of this behavior when passing a prop. Could you fix this in the Component itself?

Reply:
Thanks for pointing this out! I fixed it by removing the inline styling and added children selectors.